fractions/microprofile/microprofile-metrics/src/main/java/org/wildfly/swarm/microprofile/metrics/deployment/cdi/AMetricRegistryFactory.java

```diff
@@ -15,7 +15,7 @@
  *   limitations under the License.
  *
  */
-package org.wildfly.swarm.microprofile.metrics.cdi;
+package org.wildfly.swarm.microprofile.metrics.deployment.cdi;
```

Comment:
Can we just make the packages `deployment` and not `deployment.cdi`?

Reply:
yes, makes sense